fellowship.js

```diff
@@ -19,30 +19,63 @@ var fellowship = {
   makeMiddleEarth: function(lands) {
     console.log(lands)
     // create a section tag with an id of middle-earth
+    var earth = document.createElement("section");
+    earth.id="middle-earth";
```

Comment:
make sure to be consistent with spacing around equal signs, the preferred form is:

`var a = "foobar";`

(e.g. one space before and after the equal)